docs/pages/examples.js

```diff
@@ -53,7 +53,7 @@ class ExamplesLandingPage extends React.PureComponent {
         const gettingStartedSection = (
             <div className="">
                 <div className='prose'>
-                    <h1 className='mt24 mt0-mm' id='getting-started'>Getting started</h1>
+                    <h1 className='mt24 mt0-mm txt-fancy' id='getting-started'>Getting started</h1>
```

Comment:
I'm not sure about using `txt-fancy` here.

Reply:
To keep the headings consistent, what do you think of making "Examples" the h1 on this page and dropping "Getting started" to h2:

![image](https://user-images.githubusercontent.com/2180540/52420010-39e5ea80-2abf-11e9-8a56-0436f37de4eb.png)

This would follow the pattern of the Plugins page:

![image](https://user-images.githubusercontent.com/2180540/52420062-4e29e780-2abf-11e9-9037-2e89424cb81f.png)

Reply:
Yes. I think that makes more sense -- I guess I think of `h1` as describing the whole page and `Examples` is more accurate than `Getting started` if we follow that guideline. 🙂